chainbase/src/main/java/org/tron/core/db2/core/SnapshotManager.java

```diff
@@ -493,7 +493,8 @@ public void check() {
     if (!isV2Open()) {
       List<String> cpList = getCheckpointList();
       if (cpList != null && cpList.size() != 0) {
-        logger.error("checkpoint check failed, can't convert checkpoint from v2 to v1");
+        logger.error("checkpoint check failed, can't convert checkpoint from v2 to v1," +
+            " please set checkpoint.version = 2 in your config file and restart it.");
```

Comment:
how and where for config if `checkpoint.version` is missed ?

Reply:
Maybe a full config path is better: `storage.checkpoint.version`

Reply:
> how and where for config if `checkpoint.version` is missed ?

The full path is needed.

Reply:
change `and restart it` to `and restart` or `and restart the node`?

Reply:
Change to ''and restart the node'' seems to be more precise.